Products/ZenModel/migrate/addOOMParams.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2019, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+import Migrate
+import servicemigration as sm
+
+from Products.ZenModel.ZMigrateVersion import SCHEMA_MAJOR, SCHEMA_MINOR, SCHEMA_REVISION
+from Products.ZenUtils.controlplane.client import getCCVersion
+
+log = logging.getLogger("zen.migrate")
+sm.require("1.1.13")
+
+
+class AddOOMParams(Migrate.Step):
+    "Add OOMKillDisable and OOMScoreAdj parameters to db services"
+
+    version = Migrate.Version(SCHEMA_MAJOR, SCHEMA_MINOR, SCHEMA_REVISION)
+
+    def cutover(self, dmd):
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        service_names = ['mariadb-model', 'redis', 'RegionServer', 'ZooKeeper',
+                         'HMaster', 'Impact', 'Solr', 'mariadb-events']
+        services = filter(lambda s: s.name in service_names, ctx.services)
+        log.info("Found %i services" % len(services))
```

Comment:
Small nitpick, but something you should get into the habit of doing.  Python logging can defer interpolating the string for the log message until it's written.  However, to enable this, you can't use the `%` operator and instead, you must pass the interpolation values as arguments to the log method, e.g.
```
log.info("Found %i services", len(services))
```